### PR TITLE
Pic 2208

### PR DESCRIPTION
**Pull request by rschallapalli**

### CODE REVIEW

File: src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/OffenceRequestResponse.java [hunk 1]
```diff
@@ -14,7 +16,7 @@
 public class OffenceRequestResponse {
     @NotBlank
     private final String offenceTitle;
-    @NotBlank
+    @NotNull
     private final String offenceSummary;
```
[the remark this answers is not in the document]
Comment (rschallapalli) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/OffenceRequestResponse.java:20]: Being it is a standard library annotation and its functionality is well established and tested within that library I believe it is overkill to add an IT test for this. And it wouldn't be feasible to IT test all of such annotated fields. The IT test pack would be bloated.

[the remark this answers is not in the document]
Comment (vb-moj) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/OffenceRequestResponse.java:20]: Sorry, My comment about adding an IT is not for the annotation but for this AC.
`Expected Result 201 Created returned Actual Result 400 Bad Request`

[the remark this answers is not in the document]
Comment (jhevans) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/OffenceRequestResponse.java:20]: I'd tend to agree with @rschallapalli on this one though I also agree that in principle if you've fixed a bug, there should be a test for the case that was previously missed. 

We could create a test that iterates over a list of expected validation failures to make it a bit more tractable. What you'd end up with though is just a list of each of the annotations you've applied... you'd get protection from regressions, documentation.. but I'd argue the validation code is already self documenting. Regressions aren't a huge issue either as it's pretty static code.

I'd be happy to accept as is, validation tests might be a good topic for discussion at the next catch up though.

[the remark this answers is not in the document]
Comment (vb-moj) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/OffenceRequestResponse.java:20]: Sorry for not being very clear in my comment @jhevans & @rschallapalli .  
To make myself clear again. I didn't ask for any test to validate the annotation changes etc. 

My comment is we should have an IT to verify the change in behaviour (`Expected Result 201 Created returned Actual Result 400 Bad Request`) of the system. From my point of view, All AC must have an IT.

[the remark this answers is not in the document]
Comment (vb-moj) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/OffenceRequestResponse.java:20]: If you guys are ok with it then it's fine but it is an industry-standard that ACs should have an IT if possible!

**Review by jhevans — APPROVED** [review]

**Pull request merged**